PrefectHQ/prefect
# Wrap Prefect Module Path Parent in Quotes to Fix Windows Path Issues

Closes https://github.com/PrefectHQ/prefect/issues/6821 
This PR wraps the path to Prefect module parent in double quotes to allow spaces in the path name when starting Orion on windows. 

QA'd by: 
- running `prefect orion start` on a Window virtual machine running Windows 11 Home, with a space in the path to Prefect
- running `prefect orion start` on an M1 mac to confirm that it still runs
<img width="1362" alt="Screen Shot 2022-10-18 at 12 58 37 PM" src="https://user-images.githubusercontent.com/44583861/196497361-18eb683e-c861-4691-8545-65964e7b0b24.png">




### Example


### Checklist


	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
